tsconfig.json

```diff
@@ -10,7 +10,7 @@
       "es2017.object",
       "dom"
     ],
-    "target": "es5",
+    "target": "es6",
```

Comment:
Not strictly necessary however ES6 code is smaller then ES5 (2.8MB vs 3.1MB) so your hard drives won't work that hard and you probably save 1ms here :)